### ISSUE TITLE
Comments about Table1 (from CFRG members)

### ISSUE THREAD

**Issue by yumi-sakemi** [cfrg/draft-irtf-cfrg-pairing-friendly-curves#45]
For request from Watson

- Table 1 is very large, and it reduces the readability of the draft,
so we will consider whether we can devise an expression.
(For example, we move Table1 to Appendix and refer it)

**Comment by yumi-sakemi** [comment]
Based on the comment from Watson, we considered whether Table1 is removed or moved to Appendix about handling of Table1, however we found that it is so difficult to do.

One of our selection policies is "widely used".
Table 1 shows the adoption status of pairing-friendly curves and it is referred as an evidence to show that the selected curves matches the policy "widely used".
Therefore, Table1 is indispensable as a component of our draft, and if Table1 is excluded, the validity of the draft will be lost.
We're very grateful for the comments from Watson, but we decided to leave Table 1 in the main texts for readability of the draft.
